cmd/notify-mailer/main_test.go

```diff
@@ -15,6 +15,15 @@ import (
 	"github.com/letsencrypt/boulder/test"
 )
 
+func TestUniq(t *testing.T) {
+	input := []string{"c", "d", "c", "c", "e", "d", "e", "e"}
+	expected := "[c d e]"
+	output := uniq(input)
+	if fmt.Sprintf("%s", output) != expected {
```

Comment:
It feels slightly less hacky to use `reflect.DeepEqual` or to range over `expected` and compare vs casting the slice to a string and comparing the string representation. Is there a reason to prefer the string method here I missed?